packages/shared/src/types/abTests/banner.ts

```diff
@@ -18,13 +18,19 @@ import { PurchaseInfoTest } from './shared';
 import { z } from 'zod';
 import { OphanComponentType, OphanProduct } from '@guardian/libs';
 
+/**
```

Comment:
I think it's fair to say this is not the model we'd have chosen if building this from scratch. But it was the result of a backwards-compatible migration towards the `DesignableBanner`

Reply:
could we move the sign in prompt banner to a design and make the model simpler?

Reply:
that would be ideal! It does look quite different currently, so I think it's a question for another day